src/prefect/run_configs/docker.py

```diff
@@ -30,7 +33,13 @@ class DockerRun(RunConfig):
     """
 
     def __init__(
-        self, *, image: str = None, env: dict = None, labels: Iterable[str] = None
+        self,
+        *,
+        image: str = None,
+        env: dict = None,
+        labels: Iterable[str] = None,
+        host_config: dict = {}
```

Comment:
```suggestion
        host_config: dict = None
```
This will raise complaints in our static analysis; you'll want to set the default in the __init__ method body. I'd be consistent with whatever is being done with `env`